core/libcamera_app.cpp

```diff
@@ -381,7 +381,8 @@ void LibcameraApp::ConfigureVideo(unsigned int flags)
 	// Now we get to override any of the default settings from the options_->
 	StreamConfiguration &cfg = configuration_->at(0);
 	cfg.pixelFormat = libcamera::formats::YUV420;
-	cfg.bufferCount = 6; // 6 buffers is better than 4
+	if (options_->buffer_count > 0)
+		cfg.bufferCount = options_->buffer_count;
```

Comment:
I think this logic might be a bit wrong.  Do we instead want to do something like:

```
cfg.bufferCount = 6; // 6 buffers is better than 4
if (options_->buffer_count > 0)
    cfg.bufferCount = options_->buffer_count;
```

Ditto for the change below.

Reply:
It's better to explicitly set the value 6 as before.  I'm not sure what the pipeline handler defaults to, but if that number changes in the future, it should not affect libcamera-apps if we keep this value at a default of 6.

Reply:
For what it defaults to, chek this for raspberry pi cameras. https://github.com/raspberrypi/libcamera/blob/7c855784a28d0c9b742c6137bf311ac3779373df/src/libcamera/pipeline/raspberrypi/raspberrypi.cpp#L600

What if I change the default of the options to 6 inset of 0? t
This way we keep the current behavior, and give the option of the user to use the original defaults

Reply:
> What if I change the default of the options to 6 inset of 0? t

That would be fine with me.